collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -898,14 +898,6 @@ protected void onActivityResult(int requestCode, int resultCode, final Intent in
             case RequestCodes.AUDIO_CHOOSER:
             case RequestCodes.VIDEO_CHOOSER:
             case RequestCodes.IMAGE_CHOOSER:
-                /*
```

Comment:
Guessing this just isn't relevant anymore?

Reply:
That's right -- that code has changed a lot and we're way past Android 1.6!